src/commands/kv/bucket/mod.rs

```diff
@@ -60,9 +62,12 @@ pub fn directory_keys_values(
 }
 
 // Returns only the hashed keys for a directory's files.
-fn directory_keys_only(directory: &Path) -> Result<Vec<String>, failure::Error> {
+fn directory_keys_only(target: &Target, directory: &Path) -> Result<Vec<String>, failure::Error> {
     let mut upload_vec: Vec<String> = Vec::new();
-    for entry in WalkDir::new(directory) {
+
+    let dir_walker = get_dir_iterator(target, directory)?;
+
+    for entry in dir_walker {
```

Comment:
what can an `entry` be? mostly just curious

Reply:
Entry is any item found within a directory, whether it is a dir or file. it also can include symlinks.